### DISCUSSION TITLE
Does gotenberg have any built-in support for authorization?

### DISCUSSION THREAD

**Discussion by jhford-scout24** [gotenberg/gotenberg#851] [Q&A]
I would like to deploy Gotenberg to an application hosting platform.  The platform does not have any network level security (e.g. all containers have an internet reachable port) and as a result, I want to make sure that only requests from our other software is able to run requests against this instance.  

Does gotenberg have any built in authorization mechanisms?  I couldn't find anything in the documentation.  I'd also be interested to hear any other strategies which have been used in similar environments.

**Comment by gulien** [comment]
Hello @jhford-scout24,

Currently not. The issue has already been reported here: https://github.com/gotenberg/gotenberg/issues/684. Implementation should be easy for basic auth I think.

**Reply by jhford-scout24** [reply]
If this is something which would be accepted as a contribution I might have a chance to implement it. Would something like basic auth be acceptable?  What about having different strategies for various vendors?
